python/drned_xmnr/op/base_op.py

```diff
@@ -150,17 +150,20 @@ def setup_drned_env(self, trans):
         env = dict(os.environ)
         root = maagic.get_root(trans)
         drdir = root.drned_xmnr.drned_directory
-        if drdir is None:
+        if drdir == "env":
             try:
                 drdir = env['DRNED']
             except KeyError:
                 raise ActionError('DrNED installation directory not set; ' +
                                   'set /drned-xmnr/drned-directory or the ' +
                                   'environment variable DRNED')
+        elif drdir == "builtin" or drdir is None:
```

Comment:
I think this deserves at least `tailf:info` change for the leaf, or even better remodeling it a little bit, so that the user knows what to expect without looking into the code; the same for the magic string `"env"` above.

Reply:
Did not add the intended YANG model changes . Added and committed the missing YANG model changes.